binding/android/Falcon/falcon/src/main/java/ai/picovoice/FalconSegments.java

```diff
@@ -0,0 +1,92 @@
+/*
+    Copyright 2024 Picovoice Inc.
+
+    You may not use this file except in compliance with the license. A copy of the license is
+    located in the "LICENSE" file accompanying this source.
+
+    Unless required by applicable law or agreed to in writing, software distributed under the
+    License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+    express or implied. See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+package ai.picovoice.falcon;
+
+/**
+ * FalconSegments Class.
+ */
+public class FalconSegments {
+
+    private final Segment[] segmentArray;
+
+    /**
+     * Constructor.
+     *
+     * @param segmentArray Diarized segments and their associated metadata.
+     */
+    public FalconSegments(Segment[] segmentArray) {
+        this.segmentArray = segmentArray;
+    }
+
+    /**
+     * Getter for diarized segments and their associated metadata.
+     *
+     * @return Diarized segments and their associated metadata.
+     */
+    public Segment[] getSegmentArray() {
+        return segmentArray;
+    }
```

Comment:
if this part is just handling the array, we dont really need to create a class for it. Can just put the class below call it `FalconSegment`. Other parts of the code is fine using it as `FalconSegment[]` for the array.